qa/rpc-tests/key_import_export.py

```diff
@@ -40,11 +39,8 @@ def alice_to_bob(amount):
 
         def verify_utxos(node, amounts):
             utxos = node.listunspent(1, 10**9, [addr])
-
-            def cmp_confirmations_high_to_low(a, b):
-                return cmp(b["confirmations"], a["confirmations"])
-
-            utxos.sort(cmp_confirmations_high_to_low)
+            utxos.sort(key=lambda x: x["confirmations"])
+            utxos.reverse()
```

Comment:
This [could be](https://docs.python.org/3/library/stdtypes.html#list.sort) `utxos.sort(key=lambda x: x["confirmations"], reverse=True)`.